buildbot/check.py

```diff
@@ -5,7 +5,7 @@
 
 DEFAULT_CPU_COUNT = 4
 
-def do_check(args):
+def do_check_all_lit(args, cpu_count):
```

Comment:
What is the point of the second argument? Anyway it is overwritten at line 11

Reply:
:man_facepalming: good catch!

Reply:
Fixed.